### DISCUSSION TITLE
How to change backgroundColor?

### DISCUSSION THREAD

**Discussion by WenLonG12345** [icflorescu/mantine-datatable#195] [Q&A]
Hi, thanks for this awesome library. But I found that we cannot modify the backgroundColor for the DataTable. 
Here is my approach

```
          <DataTable<TopWinner>
            style={{
              background: "white",
              backgroundColor: "white",
              borderRadius: 5,
            }}
          />
```

But unfortunately, the background color still the same. Any idea? Thanks in advanced!

**Comment by bastiengrignon** [comment]
I'v tried to do the same for the headers using `classNames` and `createStyles` from Mantine but my header is still white. When inspecting the DOM I've seen that there is a `backgroundColor: #fff` applied to all `<tr>` tag that is overriding my custom backgroundColor ! 
How can I remove the style of this `<tr>` tag ? @icflorescu
